src/api/app/models/bs_request_permission_check.rb

```diff
@@ -39,7 +39,7 @@ class ReviewNotSpecified < APIException;
 
   def check_accepted_action(action)
     unless @target_project
-      raise NotExistingTarget.new "Unable to process project #{action.target_project}; it does not exist."
+      raise NotExistingTarget, "Unable to process project #{action.target_project}; it does not exist."
```

Comment:
Can you rephrase your commit message?
```
 [api][webui] minor cleanup 
```
is not really self-explanatory ... (Maybe just use the description?)